source/Grid/Grid.test.js

```diff
@@ -120,43 +120,49 @@ describe('Grid', () => {
   describe('shows and hides scrollbars based on rendered content', () => {
     const scrollbarSize = getScrollbarSize()
 
+    function renderTwice (markup) {
```

Comment:
Because in first render, `overflowX` or `overflowY` always won't be set
(because `this._scrollbarSize` was `undefined`)